nixos/modules/services/networking/zeronsd.nix

```diff
@@ -0,0 +1,119 @@
+{ config, lib, pkgs, ... }:
+
+let
+  cfg = config.services.zeronsd;
+  settingsFormat = pkgs.formats.json {};
+  inherit (lib) mkOption;
+in {
+  options.services.zeronsd.enable = lib.mkEnableOption (lib.mdDoc "ZeroNSD");
+
+  options.services.zeronsd.servedNetworks = mkOption {
+    default = [];
+    example = [ "a8a2c3c10c1a68de" ];
+    type = lib.types.listOf lib.types.str;
+    description = lib.mdDoc "List of ZeroTier Network IDs to start zeronsd instance for.";
+  };
+
+  options.services.zeronsd.settings = lib.mkOption {
```

Comment:
This (and more): You can either `lib.mkOption` or `inherit (lib) mkOption`, but why do both?